src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt

```diff
@@ -157,4 +159,9 @@ class PaymentRequestsService(
   }
 
   fun getDueDateString(date: XMLGregorianCalendar?): String? = date?.toString()
```

Comment:
remove unused function `getDueDateString`

Reply:
Fixed in 524b21a